### ISSUE TITLE
Fix NEI hiding unrelated BOP food/misc items when hiding items via CT scripts.

### ISSUE THREAD

**Issue by MagmaBro123** [embeddedt/ArchaicFix#40]
Title.

![image](https://user-images.githubusercontent.com/51277766/192326550-edbb5b0b-2b2a-4c4d-a8f4-dff48f3c5924.png)

See Discord.

https://canary.discord.com/channels/741043720241152001/993717703032447016/1023684699090075658

**Comment by embeddedt** [comment]
Does this occur without ArchaicFix installed?

**Comment by MagmaBro123** [comment]
> Does this occur without ArchaicFix installed?

Yes, see the modlist.

**Comment by tyrael93** [comment]
Looks more like a NHEI thing. Have you tried with regular run of the mill NEI?

**Comment by MagmaBro123** [comment]
Yes, tried both normal and HNEI. Also tried MineTweaker and CraftTweaker. Same issue.
